source/crud/tests/README.rst

```diff
@@ -61,6 +61,14 @@ Each YAML file has the following keys:
     and "sharded". If this field is omitted, the default is all topologies (i.e.
     ``["single", "replicaset", "sharded"]``).
 
+  - ``serverlessMode``: (optional) Whether or not the test should be run on
```

Comment:
This was modeled after the "sslMode" option from the server, let me know if it the terms sound okay. It's the same exact field in the unified format.